wpeverest/everest-forms
# Fix - Load user filled value in likert field in form submission when error is occurred.

### All Submissions:




### Changes proposed in this Pull Request:

Load previous likert field value in form submission when error is occurred.


### How to test the changes in this Pull Request:

1. Create a form with at least one likert field.
2. And submit  form if error occurred in any  other field. 
3. Verify it preserves user filled value in likert field or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Load user filled value in likert field in form submission when error is occurred.
